exercises/binary-search-tree/rotational-cipher/build.gradle

```diff
@@ -0,0 +1,17 @@
+apply plugin: "java"
```

Comment:
I don't think you meant to add all the rotational-cipher files in the binary-search-tree directory :)